src/test/scala/io/iohk/ethereum/jsonrpc/EthServiceSpec.scala

```diff
@@ -1221,7 +1146,7 @@ class EthServiceSpec
 
     val jsonRpcConfig = JsonRpcConfig(Config.config, available)
 
-    val ethService = new EthService(
+    lazy val ethService = new EthService(
```

Comment:
Why? I think we should avoid laziness when starting an app (to fail fast if sth is missing)